src/acquire.tsx

```diff
@@ -31,30 +27,62 @@ interface IProps extends RouteComponentProps {
     plan: IPlanObject;
 }
 
-class AcquirePage extends React.Component<IProps> {
+interface IState {
+    planId: number;
+    onPlanChange: (planId: number) => void;
+}
+
+class AcquirePage extends React.Component<IProps, IState> {
+    public constructor(props: IProps) {
+        super(props);
+        this.state = {
+          planId: -1,
+          onPlanChange: this.handlePlanChange
+        };
+      }
     render() {
         return (
           <Container maxWidth="sm">
           <Box my={4}>
             <Typography variant="h4" component="h1" gutterBottom>
               This is where we will acquire data...
-              <button onClick={this.handleSubmitClick}>Submit</button>
               <button onClick={this.handleClearQueue}>Clear Queue</button>
             </Typography>
               <Typography variant="h6" component="h1" gutterBottom>
               <div>
                   loading: {this.props.loading}.
               </div>
             </Typography>
+            <FormControl>
+                <InputLabel id="demo-simple-select-label">type</InputLabel>
+                <Select
+                    labelId="demo-simple-select-label"
+                    id="demo-simple-select"
+                    value={this.state.planId}
+                    onChange={this.handleChange}
+                >
+                    <MenuItem value={0}>count</MenuItem>
+                    <MenuItem value={1}>scan</MenuItem>
+                </Select>
+            </FormControl>
+            <Button variant="contained" onClick={this.handleSubmitClick}>Submit</Button>
```

Comment:
```suggestion
            <Button variant="contained" onClick={this.handleSubmitClick}>Submit plan to queue</Button>
```

Reply:
Happy to for now, but I think it will become clearer once it grows and the interface is improved. I would say in general sticking to one or two words for buttons is best, I intend to work on layout a little more next so that we have a plan area where you select one, and then you fill in some of the variables if you want to. Just trying to figure out the components and layouts - it may start looking useful/real early next week!